src/utils.ts

```diff
@@ -1,10 +1,11 @@
 import { createMimeMessage } from 'mimetext'
-import PostalMime from 'postal-mime'
 
 export async function sendEmail(
```

Comment:
Intended to refactor this into an example `EmailAgent` base class but it got late